draft-ietf-quic-transport.md

```diff
@@ -367,6 +367,10 @@ version negotiation and 1-RTT keys are established.
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                          Payload (*)                        ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                        [FNV Hash(64)]                         +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
Isn't the FNV1 hash expected to be appended to the packet (not the header)?

[Section 6.1 of QUIC-TLS](https://quicwg.github.io/base-drafts/draft-ietf-quic-tls.html#rfc.section.6.1) states:
> The sender then calculates the integrity check over the entire packet, starting from the type field. The output of the hash is appended to the packet.